### ISSUE TITLE
`@stylistic/indentation` asks for minus one tab on a multi-line set of media parameters

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#237]
Found while triaging the `indentation` rows of the control oracle written for the plan of 2026-08-20. It is not a comment bug: the same file without any comment draws the same warning.

### What minimal example or steps are needed to reproduce the bug?

A media query whose closing parenthesis stands on a line of its own:

```scss
@media (min-width: 100px
	) { a { color: red; } }
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "customSyntax": "postcss-scss",
  "rules": {
    "@stylistic/indentation": "tab"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.scss" --config myconfig.json
```

### What did you expect to happen?

Either an expectation a file can satisfy, or nothing.

### What actually happened?

```
2:2 Expected indentation of -1 tabs
```

There is no such indentation to write. Whatever the rule meant to ask for, minus one tab is a count that fell through arithmetic rather than a level.

### What is the cause?

Not established. The level of a continuation line inside a set of at-rule parameters is worked out by counting parentheses and subtracting, and on this shape the subtraction goes one past zero.
